error CS1503: Argument 1: cannot convert from 'Microsoft.UI.Xaml.Controls.NavigationView' to 'Uno.Toolkit.UI.DrawerControl'

## On which page?
https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/controls/DrawerControl.html#navigationview-with-drawer
## What's wrong?
error CS1503: Argument 1: cannot convert from 'Microsoft.UI.Xaml.Controls.NavigationView' to 'Uno.Toolkit.UI.DrawerControl' when trying to compile that code
## Any feedback?
removing `utu:DrawerControlBehavior.FitToDrawerContent="False"` makes the code compile fine

--- a/build/workflow/cspell.json
+++ b/build/workflow/cspell.json
@@ -2,6 +2,7 @@
 	"version": "0.2",
 	"language": "en",
 	"words": [
+		"autoplay",
 		"Avalonia",
 		"ambiently",
 		"binlog",

--- a/doc/helpers/StatusBar-extensions.md
+++ b/doc/helpers/StatusBar-extensions.md
@@ -17,7 +17,6 @@ Provides two attached properties on `Page` to control the visuals of the status 
     </iframe>
 </div>
 
-
 ## Remarks
 
 The attached properties do nothing on platforms other than iOS and Android.

--- a/src/Uno.Toolkit.UI/Behaviors/DrawerControlBehavior.cs
+++ b/src/Uno.Toolkit.UI/Behaviors/DrawerControlBehavior.cs
@@ -41,10 +41,10 @@ namespace Uno.Toolkit.UI
 			new PropertyMetadata(default(Brush)));
 
 		[DynamicDependency(nameof(SetDrawerBackground))]
-		public static Brush GetDrawerBackground(DrawerControl obj) => (Brush)obj.GetValue(DrawerBackgroundProperty);
+		public static Brush GetDrawerBackground(DependencyObject obj) => (Brush)obj.GetValue(DrawerBackgroundProperty);
 
 		[DynamicDependency(nameof(GetDrawerBackground))]
-		public static void SetDrawerBackground(DrawerControl obj, Brush value) => obj.SetValue(DrawerBackgroundProperty, value);
+		public static void SetDrawerBackground(DependencyObject obj, Brush value) => obj.SetValue(DrawerBackgroundProperty, value);
 
 		#endregion
 		#region DependencyProperty: OpenDirection
@@ -127,9 +127,9 @@ namespace Uno.Toolkit.UI
 			new PropertyMetadata(DrawerControl.DefaultValues.FitToDrawerContent));
 
 		[DynamicDependency(nameof(SetFitToDrawerContent))]
-		public static bool GetFitToDrawerContent(DrawerControl obj) => (bool)obj.GetValue(FitToDrawerContentProperty);
+		public static bool GetFitToDrawerContent(DependencyObject obj) => (bool)obj.GetValue(FitToDrawerContentProperty);
 		[DynamicDependency(nameof(GetFitToDrawerContent))]
-		public static void SetFitToDrawerContent(DrawerControl obj, bool value) => obj.SetValue(FitToDrawerContentProperty, value);
+		public static void SetFitToDrawerContent(DependencyObject obj, bool value) => obj.SetValue(FitToDrawerContentProperty, value);
 
 		#endregion
 	}

